app/services/construct_cap_attributes.rb

```diff
@@ -16,13 +16,11 @@ def perform
         purchase_type: ConstructCapAttributes::PURCHASE_TYPE,
         product_name_and_description: product_name_and_description,
         justification: justification,
-        link_to_product: link_to_product,
         cost_per_unit: cost_per_unit,
         quantity: ConstructCapAttributes::QUANTITY,
         recurring: ConstructCapAttributes::RECURRING,
         date_requested: date_requested,
-        urgency: ConstructCapAttributes::URGENCY,
-        additional_info: additional_info
+        urgency: ConstructCapAttributes::URGENCY
```

Comment:
Is there a specific reason those fields were removed here? Not complaining, just wondering.